tests/test_internals/test_roi_extractors_module.py

```diff
@@ -230,6 +242,243 @@ def test_add_two_imaging_planes(self):
         assert second_imaging_plane.description == second_imaging_plane_description
 
 
+class TestAddImageSegmentation(unittest.TestCase):
+    def setUp(self):
```

Comment:
One more thing,  this is up to you but something that you can do in testing is that variables that are common to all tests such as the session_start_time can be defined in a ``setUpClass(cls)` method at the beginning of the TestCase. See this for an example:

https://github.com/catalystneuro/neuroconv/blob/36ab8d2d21d7a0c2932fc6f849807fbdc60cb40b/tests/test_internals/test_si.py#L609-L615

Reply:
Ah you're right, I'll restructure the setup.
Thank you @h-mayorquin for the comments!